easybuild/easyconfigs/f/FLANN/cmake-flann-1.9.1.patch

```diff
@@ -0,0 +1,21 @@
+diff -Nru flann-1.9.1.orig/src/cpp/CMakeLists.txt flann-1.9.1/src/cpp/CMakeLists.txt
```

Comment:
@sassy-crick this patch is identical to the one above? no need for this one then. it does not matter that the name of the directory is different.

Reply:
They do the same thing but the name of the directory is different (flann-1.8.4 vs. flann-1.9.1). How would that work?

Reply:
the name of the directory is ignored. just try it, you will see that it works ;)

Reply:
Thanks for that. Something learned. I have merged the two patch files into one as they are the same now. That means I have changed the name to make it version independent. I also changed the two EC files accordingly..